packages/system/cozystack-api/templates/deployment.yaml

```diff
@@ -21,6 +21,8 @@ spec:
       labels:
         app: cozystack-api
     spec:
+      tolerations:
+      - operator: Exists
```

Comment:
This will tolerate nodes with unconfigured network, however it won't start pod until CNI is ready.
Then it seems ok